### ISSUE TITLE
AutoTuner times ignore rebuilding of verlet lists in first iteration

### ISSUE THREAD

**Issue by SteffenSeckler** [AutoPas/AutoPas#120]
**Describe the bug**
When tuning while rebuilding in every step, in the first iteration of container2 the rebuilding of the verlet lists is ignored. Please don't ignore it.
```
[2018-11-20 16:44:05.650] [AutoPasLog] [debug] Updating container
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] ContainerSelector: Collected container times:
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 0 took 78026793 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 0 took 84209310 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 0 took 81558400 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 1 took 28485782 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 1 took 28232811 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 1 took 27331027 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 1 took 27865813 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 1 took 28592472 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 1 took 28404855 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 2 took 13937330 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 2 took 29776190 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Container 2 took 29488909 nanoseconds:   
[2018-11-20 16:44:05.651] [AutoPasLog] [debug] Selected container 2
```

**Expected behavior**
Is not ignored. Also:
Proper calculation or measurement of enough samples + mean value should be taken.

**Additional context**

**Comment by FG-TUM** [comment]
these should be two issues / feature requests :S

**Comment by FG-TUM** [comment]
fixed by #122
